articles/email/providers.md

```diff
@@ -1,60 +1,119 @@
-# Using your own SMTP Provider for sending emails
+# Use your own SMTP Email Provider
 
-Instead of using Auth0's infrastructure for email communication with your users the Auth0 API v2 also allows you to configure your own provider. This allows you to better monitor and troubleshoot the email communication.
+The Auth0 API v2 allows you to configure your own SMTP email provider instead of using Auth0's infrastructure for email communication with your users. This enables you to more completely manage, monitor and troubleshoot your email communications.
```

Comment:
I would remove the "Auth0 API v2" bit. You can configure it via the dashboard or the API - either way the "how" you actually do the configuration isnt really relevant in the intro.